packages/frontend/src/app/create-store/page.tsx

```diff
@@ -8,11 +8,11 @@ import React, { useState, useEffect, useRef } from "react";
 import AvatarUpload from "@/app/common/components/AvatarUpload";
 import { useMyContext } from "@/context/MyContext";
 import { useAuth } from "@/context/AuthContext";
-import * as abi from "@massmarket/contracts";
```

Comment:
just removing some code so frontend  build passes for now with the blockchain change